tests/app/clients/test_dvla.py

```diff
@@ -156,3 +171,232 @@ def test_jwt_token_calls_authenticate_if_expiry_time_passed(client, dvla_client,
         assert dvla_client._jwt_expires_at == one_hour_later
 
     assert mock_authenticate.called_once
+
+
+def test_format_create_print_job_json_builds_json_body_to_create_print_job(dvla_client):
```

Comment:
It might be nice to have these as class-based tests, eg:

```python
class TestFormatCreatePrintJobJson:
    def test_builds_expected_json(self):
        ...
```

Since the next few tests all have the same prefix. It's vaguely preference, but it does provide good and clear encapsulation of tests for a given thing

Reply:
I prefer non-classed based testing - I'll just stick with the current way for now since it's more consistent with how we write tests in the rest of the code base

Reply:
👍